tbx/project_styleguide/templates/patterns/organisms/header/header.html

```diff
@@ -47,27 +47,31 @@
     <div class="grid">
         <div class="header__bottom grid__header-nav">
             <div class="header__logo">
-                {% include "patterns/atoms/icons/icon.html" with name="logo" classname="logo" %}
-                {% if page.final_division and page.final_division.label %}
-                    <span class="logo__label">
-                        {{ page.final_division.label }}
-                    </span>
+                {% if page.final_division and page.final_division.logo %}
+                    {% with class_name="logo logo--"|add:page.final_division.logo %}
```

Comment:
It seems we're using both `logo--logo-torchbox` and `logo--torchbox` classes in different cases, while we only have styles for `logo--torchbox`. 
![image](https://github.com/user-attachments/assets/777502fa-d1c5-4fcb-910a-119f0519359a)

Reply:
thanks, I've updated this